qa/workunits/ceph-disk/ceph-disk-test.py

```diff
@@ -202,8 +218,43 @@ def test_destroy_osd(self):
         partition = c.get_osd_partition(osd_uuid)
         assert partition['type'] == 'data'
         assert partition['state'] == 'active'
-        c.destroy_osd(osd_uuid)
+        c.sh("ceph-disk deactivate " + partition['path'])
```

Comment:
wait for osd down (or something similar) as well here to assert the desired side effect of **deactivate**